server/storage/sql_models.go

```diff
@@ -50,7 +50,7 @@ func (c Change) TableName() string {
 // CreateTUFTable creates the DB table for TUFFile
 func CreateTUFTable(db gorm.DB) error {
 	// TODO: gorm
-	query := db.Set("gorm:table_options", "ENGINE=InnoDB DEFAULT CHARSET=utf8").CreateTable(&TUFFile{})
+	query := db.AutoMigrate(&TUFFile{})
```

Comment:
The switch to sqlite in a couple of tests mean the `ENGINE` options is no longer valid.